### DISCUSSION TITLE
Can I specify a custom instanceId when starting a SubOrchestrator?

### DISCUSSION THREAD

**Discussion by marked23** [Azure/azure-functions-durable-extension#2710] [Q&A]
Can I specify a custom instanceId when starting a SubOrchestrator?

**Comment by davidmrdavid** [comment]
Indeed! SubOrchestrators have an `instanceId` parameter, see here: https://learn.microsoft.com/en-us/dotnet/api/microsoft.azure.webjobs.extensions.durabletask.idurableorchestrationcontext.callsuborchestratorasync?view=azure-dotnet#microsoft-azure-webjobs-extensions-durabletask-idurableorchestrationcontext-callsuborchestratorasync(system-string-system-string-system-object)
